### PR TITLE
LPS-152488 Define Cookie Preferences test suite for CI

### PR DESCRIPTION
**Pull request by gaborlovasdotcom**
[LPS-152488](https://issues.liferay.com/browse/LPS-152488)

### CODE REVIEW

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:cookie-preferences:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
@JoyceWang08211 , Could you please review what else I miss?

**Comment by gaborlovasdotcom** [comment]
ci:test:cookie-preferences

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:forward

File: test.properties [hunk 1]
```diff
@@ -3215,6 +3215,20 @@
...
+
+    test.batch.class.names.includes[cookie-preferences]=\
+        **/cookies/**/*Test.java
+
+    test.batch.dist.app.servers[cookie-preferences]=tomcat
+
+    test.batch.names[cookie-preferences]=${test.batch.names[security]}
+
+    test.batch.run.property.query[functional-tomcat90-mysql57-jdk8][cookie-preferences]=\
+        (testray.component.names ~ "Cookie Preferences")
```
Comment (JoyceWang08211) [inline test.properties:3230]: It's better to have this PQL also:
(portal.upstream == true) AND \

It's helpful to us to run specific case in the future. But it's not a mandatory requirement.

**Pull request closed without merging**